app/components/Home.tsx

```diff
@@ -0,0 +1,270 @@
+import * as React from 'react';
+const { EditorKit, EditorKitDelegate } = require('sn-editor-kit');
+const MarkdownIt = require('markdown-it');
+
+const EditMode = 0;
```

Comment:
Should be refactored into TypeScript enums.

Reply:
What does this look like?

Reply:
https://www.typescriptlang.org/docs/handbook/enums.html

Reply:
Enums are implemented as we discussed 👍